### PR TITLE
refactor: update all layout styles

### PR DESCRIPTION
**Pull request by enesozturk**
# Description

Refactor layout styles to fix overflow issues in different situations like; subscription list, notifications (topic) page on mobile, etc.

# Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

# Fixes/Resolves (Optional)

Closes https://github.com/WalletConnect/web3inbox/issues/379

# Examples/Screenshots (Optional)

[See preview for better showcase]

# Checklist:

- [ ] I have performed a self-review of my own code
- [ ] My changes generate no new warnings
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/App.scss [hunk 1]
```diff
@@ -38,11 +42,14 @@
...
     height: auto;
-    min-height: 100svh;
-    overflow-x: hidden;
+    overflow-x: auto;
 
     @media only screen and (max-width: 700px) {
-      padding-bottom: calc(env(safe-area-inset-bottom) + 48px + 2rem);
+      // padding-bottom: calc(env(safe-area-inset-bottom) + 48px + 2rem);
+
+      // update
```
Comment (devceline) [inline src/App.scss:50]: update what?

File: src/App.scss [hunk 2]
```diff
@@ -38,11 +42,14 @@
     grid-area: main;
     max-width: 100vw;
     height: auto;
-    min-height: 100svh;
-    overflow-x: hidden;
+    overflow-x: auto;
 
     @media only screen and (max-width: 700px) {
-      padding-bottom: calc(env(safe-area-inset-bottom) + 48px + 2rem);
+      // padding-bottom: calc(env(safe-area-inset-bottom) + 48px + 2rem);
```
Comment (devceline) [inline src/App.scss:48]: If we're not using it feel free to delete, not comment

File: src/index.css [hunk 3]
```diff
@@ -60,3 +60,18 @@ video {
...
+  overflow-x: hidden;
+}
+
+@media only screen and (max-width: 700px) {
+  .PageContainer {
+    overflow-x: hidden;
+    overflow-y: auto;
+    padding-bottom: 1rem;
+  }
+}
```
Comment (devceline) [inline src/index.css:77]: nit: prefer `min-width` as we want to design mobile first

File: src/App.scss [hunk 4]
```diff
@@ -13,10 +13,14 @@
   z-index: 0;
   position: relative;
 
+  height: calc(100svh - 60px - env(safe-area-inset-bottom));
```
Comment (devceline) [inline src/App.scss:16]: Where is this formula coming form

**Review by devceline — APPROVED** [review]
LGTM

**Comment by enesozturk** [comment]
Updated the height units to check if the OS supports `dvh`. This fixes and issue on Android;

Tested on;
- Web (Chrome, Firefox, Safari)
- Android simulator
- iOS simulator

**Pull request merged**